lib/bundler/cli.rb

```diff
@@ -53,6 +52,11 @@ def initialize(*args)
       end
     end
 
+    def self.forgotten_option(*args, &blk)
```

Comment:
Can i suggest `removed_option` or `deprecate_option`  instead. This confused me a bit with the CLI no longer remembering options.